### PR TITLE
S3downloader post. BCD-287

### PR DESCRIPTION
**Pull request by makarski**
An article for https://github.com/motain/s3downloader

### CODE REVIEW

**Comment by makarski** [comment]
@maximevaly, @ghophp, please, review.

**Comment by ghophp** [comment]
it looks very nice, and informative.. the only thing I would add, is, after the last section (Example 3), you could place a final section, no need of a name, just a closer paragraph, talking about what the impact on using this tool instead the default amazon s3 default panel, so it works as a conclusion to the post... right now when I look to the post it looks a lot as a documentation, this is not bad, but, it could look more as a informative piece of content, and then the documentation is at the github repository... :+1:

**Comment by makarski** [comment]
@ghophp, thanks. I will update the post.

**Comment by makarski** [comment]
@ghophp, I have written a small summary and did some other minor corrections.

**Comment by maximevaly** [comment]
:+1: looks good

Note that if you want you can also add syntax highlighting by specifying the language in code blocks:

``` bash
cd $GOPATH/src/github.com/motain/s3downloader
cp config.json.dist config.json
```

**Comment by makarski** [comment]
@maximevaly, I think i tried that with bash, but probably didn't see the difference. Will update tomorrow.

**Comment by makarski** [comment]
@maximevaly, I've updated the post with syntax highlights and changed the date.

**Comment by maximevaly** [comment]
Perfect, you can merge it then :+1:

File: _posts/2016-01-08-s3downloader.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+- 2016-01-05/171500/match-a-b-results.xml
+```
+
+#### Getting up and running
+
+In order to run the command you need to have **go** installed on your machine. You can [install](https://golang.org/dl/) **go** either from source or run an installation package.
+
+Once you have installed and configured **go**, run `go get -t -u github.com/motain/s3downloader` - this will fetch the project source and install it on your machine.
+
+In order to make sure you have correctly install the tool run the following cmd from terminal:
```
Comment (dirkaholic) [inline _posts/2016-01-08-s3downloader.md:36]: Just saw: make sure you have correctly install**ed**. ed is missing.

**Pull request merged**